cypress/integration/spawn/volume.ts

```diff
@@ -138,21 +138,22 @@ describe("Navigating to Spawn Volume page", () => {
       "I understand this volume is currently mounted to a host."
     ).click();
     cy.get(popconfirmYesClassName).click();
-    cy.dataRowKey(
-      "1de2728dd9de82efc02dc21f6ca046eaa559462414d28e0b6bba6436436ac873"
-    ).should("not.exist");
-    cy.dataCy("mounted-badge").contains("8 Mounted");
-    cy.dataCy("free-badge").contains("3 Free");
```

Comment:
Since we are only commenting out part of this test is there something we can assert against instead. Otherwise it might make sense to comment out the entire test.

Reply:
The above parts of the test do assert that the modal pops up correctly it looks like, so I think it's okay to only partially comment out.

Reply:
Discussed in person; Mohamed's going to update the test.